evap/evaluation/templates/responsibility_buttons.html

```diff
@@ -3,12 +3,16 @@
 <div class="btn-group" data-toggle="buttons">
     {% for choice in form.responsibility %}
         {% if choice.data.value == "RESPONSIBLE" %}
-            {% trans "Name will be shown next to course. Able to see all comments about all contributors of this course." as tooltip %}
+            {% trans "Name will be shown next to course. Able to see all comments about all contributors of this course. This can only be set by student representatives" as tooltip %}
         {% elif choice.data.value == "EDITOR" %}
             {% trans "Able to edit the course to prepare the evaluation." as tooltip %}
         {% elif choice.data.value == "CONTRIBUTOR" %}
             {% trans "Default value. No special rights." as tooltip %}
         {% endif %}
-        {% include "choice_button.html" with formelement=form.responsibility choice=choice tooltip=tooltip %}
+        {% if choice.data.value != "RESPONSIBLE" or can_change_responsible %}
+            {% include "choice_button.html" with formelement=form.responsibility choice=choice tooltip=tooltip disabled=False %}
+        {% else %}
+            {% include "choice_button.html" with formelement=form.responsibility choice=choice tooltip=tooltip disabled=True %}
+        {% endif %}
```

Comment:
can you invert that if? i find "disable this if it's responsible and the user cannot change responsibles" easier to parse than "enable this if it is not responsible and the user can change it"